### DISCUSSION TITLE
Have yank copy to register and clipboard

### DISCUSSION THREAD

**Discussion by danjenson** [helix-editor/helix#6351] [Q&A]
Is there anyway to yank to a register and clipboard? In neovim, I often highlight blocks of code and paste them in a REPL for testing. Is there anyway to set +y to both do space+y and regular +y?

**Comment by pascalkuthe** [comment]
The normal copy to register behaviour is handled by the `yank` command and the copy to clipboard by `yank_joined_to_clipboard` so to yank to both at once you can bind `["yank", "yank_joined_to_clipboard"] ` to a key

**Comment by danjenson** [comment]
Quick follow up -- can you automatically copy to clipboard / yank register on select without needing a hotkey?

**Reply by askreet** [reply]
I don't think so, but yank to register on select would be a nightmare I would imagine, since every movement changes the selection. For me, I often yank some text and then use movements like `w`, `f`/`t`, etc. to get to where I need to be. Along the way, I've selected text many times. (Plus, technically, hjkl create single-character selections as you move!)
